### PR TITLE
Crs 663 schedule fifteen

### PR DESCRIPTION
**Pull request by ldlharper**
We need to know if an offence is schedule 15 or not in order to calculate release dates. This is currently decided by the indicators in the `OFFENCE_INDICATORS` table. This table did not have a JPA entity definition, so I've created it and the flyway scripts (I've not done this in prison API before so if someone could double check it for me that would be great :))

### CODE REVIEW

File: src/main/resources/db/migration/data/R__9_12__OFFENCE_INDICATORS.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+INSERT INTO OFFENCE_INDICATORS (OFFENCE_INDICATOR_ID, OFFENCE_CODE, STATUTE_CODE, INDICATOR_CODE) VALUES (-1, 'RC86355', 'RC86', 'S15/CJIB');
```
Comment (ldlharper) [inline src/main/resources/db/migration/data/R__9_12__OFFENCE_INDICATORS.sql:1]: I wasn't sure of the version numbers for these flyway scripts

File: src/main/resources/db/migration/nomis/ddl/V1_412__OFFENCE__INDICATORS.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (ldlharper) [inline src/main/resources/db/migration/nomis/ddl/V1_412__OFFENCE__INDICATORS.sql:1]: This DDL is from dev nomis DB.

File: src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/sentences-and-offences-details.json [hunk 3]
```diff
@@ -41,7 +44,10 @@
       {
         "offenderChargeId": -9,
         "offenceCode": "RC86355",
-        "offenceDescription": "Cause the carrying of a mascot etc on motor vehicle in position likely to cause injury"
+        "offenceDescription": "Cause the carrying of a mascot etc on motor vehicle in position likely to cause injury",
```
Comment (joelstobart-moj) [inline src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/sentences-and-offences-details.json:47]: Love this ...

**Review by joelstobart-moj — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenceIndicator.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import java.io.Serializable;
+
+@AllArgsConstructor
+@Builder
+@Data
+@EqualsAndHashCode(of = {"offence", "indicatorCode"}, callSuper = false)
+@NoArgsConstructor
+@Entity
+@Table(name = "OFFENCE_INDICATORS")
+@ToString
```
Comment (Mjwillis) [inline src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenceIndicator.java:15]: Might not want to include `offence` in ToString - causes more loads

**Review by Mjwillis — APPROVED** [review]
Apart from ToString but can be fixed in next commit

File: src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenceIndicator.java (hunk 4 above)
Comment (ldlharper) [reply src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenceIndicator.java]: Done

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Pull request merged**